### DISCUSSION TITLE
Is this project / discussion still alive? Yes it is!

### DISCUSSION THREAD

**Discussion by Freigeist069** [mediacms-io/mediacms#1011] [General]
This project is awesome and deserves more exchange about the features and options. So, it would be great if there was a bit more discussion here again. Are you still developing and are you still active here?

Warm regards, Joerg ﻿👋🙂

**Comment by AnthonyTorreso** [comment]
I too hope so, I sent an email to the listed contact and it was answered right away, however two weeks ago I sent in a request for a paid script and nothing.  I hope so as this is an awesome piece of software.

**Reply by Freigeist069** [reply]
I have written the project team via their contact form and got no answer at all. I absolutely agree with you that MediaCMS is a wonderful and very promising project. I would love to be part of this community and I would also love to be active here in a constructive way. But if the platform for constructive discussion and exchanging ideas is dead, I don't want to spend any time installing and working on MediaCMS.

**Comment by Freigeist069** [comment]
After a brief conversation with the developer of MediaCMS, I can fortunately confirm that the project is still alive and that there will be further developments in the future. Nevertheless, I think it is important to bring a little more activity to this community.  I am definitely ready to give this project the chance to grow and would like to actively participate in it.

**Reply by QuestOAuth74** [reply]
Thanks for the welcome update, Freigeist069. It is a wonderful piece of software with various use cases. I am glad they are working on it. However, progress appears to be slow. I have decided to get the assistance of a developer to add the features I need. I am afraid, looking at the pace of progress, it might take a long while before some important features are added. I will pass by now and then to check on the progress of this project.
